packages/apps/job-launcher/client/src/providers/CreateJobPageUIProvider.tsx

```diff
@@ -8,25 +8,27 @@ export type CreateJobPageUIType = {
   step: CreateJobStep;
   payMethod: PayMethod;
   jobRequest: JobRequest;
-  reset?: () => void;
-  changePayMethod?: (method: PayMethod) => void;
-  updateJobRequest?: (jobRequest: JobRequest) => void;
-  goToPrevStep?: () => void;
-  goToNextStep?: () => void;
+  reset: () => void;
```

Comment:
If we have all these methods defined now, maybe let's change all the downstream types to not use `?.`?